examples/tv-casting-app/tv-casting-common/core/CastingApp.cpp

```diff
@@ -150,6 +158,8 @@ CHIP_ERROR CastingApp::Stop()
 
     mState = CASTING_APP_NOT_RUNNING; // CastingApp stopped successfully, set state to NOT_RUNNING
 
+    chip::Server::GetInstance().GetUserDirectedCommissioningClient()->RemoveCommissionerDeclarationHandler();
+
```

Comment:
It probably makes more sense to move this line to the spot before `chip::Server::GetInstance().Shutdown()`

Reply:
Okay